ext/strings.go

```diff
@@ -44,6 +56,42 @@ import (
 //	'hello'.charAt(5)  // return ''
 //	'hello'.charAt(-1) // error
 //
+// # Format
+//
+// Returns a new string with substitutions being performed, printf-style.
+// The valid formatting clauses are:
+//
+// `%s` - substitutes a string. This can also be used on Duration and Timestamp.
```

Comment:
Are we OK with conflating duration and timestamp with string?  This locks us out of ever using directives to determine how a duration or timestamp is formatted, but it would still be possible to pre-format the duration or timestamp as a string before passing it to format.

cc @TristonianJones

Reply:
(marking as resolved, seems everyone thinks this is fine)